tests/test_grid.py

```diff
@@ -1177,7 +1177,7 @@ def test_hide_cells(ind, struct_grid):
     assert out.HasAnyBlankCells()
 
     with pytest.raises(ValueError, match='Boolean array size must match'):
-        struct_grid.hide_cells(np.ones(10, dtype=np.bool), inplace=True)
+        struct_grid.hide_cells(np.ones(10, dtype=bool), inplace=True)
```

Comment:
This was one of two actually problematic instances in this module, but I think `dtype=bool` is always preferable to `dtype=np.bool_` and especially just `np.bool_` as a positional argument. That's why I changed all the others as well in this module